cpro/magicollections.py

```diff
@@ -0,0 +1,257 @@
+        otherbuttons_template = 'include/advancedButton'
+        redirect_after_add    = collections_settings.redirectAfterAddAccount
+
+    class EditView(_AccountCollection.EditView):
+        after_template = 'include/accountJSstarter'
+        extra_context  = collections_settings.modAccountExtraContext
+        form_class     = forms.AccountFormAdvanced
+        # TODO: Not sure how to handle the ENABLED_COLLECTIONS here
+        js_files       = ENABLED_COLLECTIONS['account']['edit'].get('js_files', []) + ['mod_account']
+
+    class ListView(_AccountCollection.ListView):
+        before_template  = 'include/beforeLeaderboard'
+        distinct         = True
+        default_ordering = '-level'
+        extra_context    = collections_settings.leaderboardExtraContext
+        filter_form      = forms.FilterAccounts
+        # TODO: Replaced by get_queryset
+        filter_queryset  = filters.filterAccounts
+        # TODO: Not sure how to handle the ENABLED_COLLECTIONS here
+        js_files         = ENABLED_COLLECTIONS['account']['list'].get('js_files', []) + ['leaderboard']
+        # TODO: Not sure if this lambda works
+        show_add_button  = lambda request: not request.user.is_authenticated()
+
+############################################################
+# Activity Collection
+
+class ActivityCollection(_ActivityCollection):
+    # TODO: Not sure if this is ported properly since it's directly taken from the old `settings.py` file
+    ACTIVITY_TAGS = [
+        ('cards', _('New Cards')),
+        ('event', _('Event')),
+        ('live', _('Live')),
+        ('comedy', _('Comedy')),
+        ('room', _('Room Decoration')),
+        ('introduction', _('Introduce yourself')),
+        ('idols', _('Idols')),
+        ('anime', _('Anime')),
+        ('cosplay', _('Cosplay')),
+        ('fanart', _('Fan made')),
+        ('merch', _('Merchandise')),
+        ('community', _('Community')),
+        ('unrelated', _('Unrelated')),
+        ('AR Idol Date', 'AR Idol Date'),
+    ]
+
+    # TODO: Not sure if this is ported properly since it's directly taken from the old `settings.py` file
+    def filterActivitiesList(queryset, parameters, request):
+        if request.user.is_superuser and 'force_old' in request.GET:
+            if 'owner_id' in request.GET:
+                return queryset.filter(owner_id=request.GET['owner_id'])
+            return queryset
+        return queryset.filter(id__gt=2600)
+
+    # TODO: Not sure if this is ported properly since it's directly taken from the old `settings.py` file
+    def filterActivities(queryset, parameters, request):
+        if request.user.is_superuser:
+            return queryset
+        return filterActivitiesList(queryset, parameters, request)
+
+    class AddView(_ActivityCollection.AddView):
+        before_save     = collections_settings.activitiesBeforeSave
+        # TODO: Replaced by get_queryset
```

Comment:
Same comment as above, anything in `filters.py` should be re-implemented using the new `MagiFiltersForm` feature.